code/html/index.html

```diff
@@ -0,0 +1,64 @@
+  <div class="rcalc-container">
+    <form name="nameCalculator" class="rcalc-calculator">
+      <table>
+        <tr>
+          <td colspan="3"><input type="text" name="nameDisplayMain" id="id-display-main" maxlength="1000" disabled></td>
+          <td><input type="button" name="nameButtonClear" class="rcalc-button-clear" value="C" onclick="clearDisplay();"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameSeven" class="rcalc-button" value="7" onclick="testZero(nameSeven.value);displayNumber(nameSeven.value);"></td>
+          <td><input type="button" name="nameEight" class="rcalc-button" value="8" onclick="testZero(nameEight.value);displayNumber(nameEight.value);"></td>
+          <td><input type="button" name="nameNine" class="rcalc-button" value="9" onclick="testZero(nameNine.value);displayNumber(nameNine.value);"></td>
+          <td><input type="button" name="nameDivision" class="rcalc-button" value="/" style="background-color: #39681c" onclick="setWarning();displayNumber(nameDivision.value);"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameFour" class="rcalc-button" value="4" onclick="testZero(nameFour.value);displayNumber(nameFour.value);"></td>
+          <td><input type="button" name="nameFive" class="rcalc-button" value="5" onclick="testZero(nameFive.value);displayNumber(nameFive.value);"></td>
+          <td><input type="button" name="nameSix" class="rcalc-button" value="6" onclick="testZero(nameSix.value);displayNumber(nameSix.value);"></td>
+          <td><input type="button" name="nameMultiplication" class="rcalc-button" value="*" style="background-color: #39681c" onclick="displayNumber(nameMultiplication.value);"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameOne" class="rcalc-button" value="1" onclick="testZero(nameOne.value);displayNumber(nameOne.value);"></td>
+          <td><input type="button" name="nameTwo" class="rcalc-button" value="2" onclick="testZero(nameTwo.value);displayNumber(nameTwo.value);"></td>
+          <td><input type="button" name="nameThree" class="rcalc-button" value="3" onclick="testZero(nameThree.value);displayNumber(nameThree.value);"></td>
+          <td><input type="button" name="nameSubtraction" class="rcalc-button" value="-" style="background-color: #39681c" onclick="displayNumber(nameSubtraction.value);"></td>
+        </tr>
+        <tr>
+          <td><input type="button" name="nameDot" class="rcalc-button" value="." onclick="displayNumber(nameDot.value);"></td>
+          <td><input type="button" name="nameZero" class="rcalc-button" value="0" onclick="testZero(nameZero.value);displayNumber(nameZero.value);"></td>
+          <td><input type="button" name="nameEqual" class="rcalc-button" value="=" onclick="result();"></td>
+          <td><input type="button" name="nameAddition" class="rcalc-button" value="+" style="background-color: #39681c" onclick="displayNumber(nameAddition.value);"></td>
```

Comment:
I don't want see attribute style in HTML ... all css will only in css files. We don't want styling pages on both places ... just on one place.  Remove style from all html and put colors etc. into css classes.

Reply:
no longer style attributes inside html, but becasuse I changed class for buttons-operators due to css styling I had to change code in rcalc-math.js also, it grown in size a lot because I have now same actions for 2 classes, it does not seem to me as perfect solution, PLEASE CLOSER CHECK ON IT